cfme/tests/candu/test_utilization_metrics.py

```diff
@@ -130,6 +134,25 @@ def query_metric_db(appliance, provider, metric, vm_name=None, host_name=None):
         metrics_tbl.id.in_(provs.subquery()))
 
 
+@pytest.fixture(params=["regions", "providers", "clusters", "hosts"])
+def entity(temp_appliance_extended_db, request, provider):
+    appliance = temp_appliance_extended_db
+    provider.appliance = appliance
+    if request.param == "regions":
+        return appliance.collections.regions.instantiate(appliance.region()[-1])
+    if request.param == "providers":
+        return provider
+    if request.param == "clusters":
+        return appliance.collections.clusters.instantiate(name=provider.data["clusters"][0],
+        provider=provider)
+    if request.param == "hosts":
+        return appliance.collections.hosts.instantiate(name=provider.data["hosts"][0].name,
+        provider=provider)
+
+
+# =========================================== Tests ===============================================
+
+
```

Comment:
Please move this fixture to cfme/fixtures/candu.py .

Reply:
@nachandr DONE